{"name": "test_17_login_invalid", "status": "passed", "description": "test that checks that the user can not login to site with invalid user phone number", "start": 1680585067808, "stop": 1680585085058, "uuid": "4391ea33-354c-49dc-8ecc-ed2f41452ef7", "historyId": "08644f5d0241d1555c3d52eb4c14d3cc", "testCaseId": "da139303ebd0e457dedf96ac4e7c9c7a", "fullName": "tests.test_login.TestLogin#test_17_login_invalid", "labels": [{"name": "severity", "value": "Critical"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "AlexanderZak"}, {"name": "thread", "value": "4068-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}